docker-hub/api/dvp.yaml

```
@@ -12,7 +12,14 @@ info:
 
      In your summary data CSV, you will have access to the data points listed below. You can request summary data for a complete week (Monday through Sunday) or for a complete month (available on the first day of the following month). 
 
-     - For every namespace, repository, and reference (tag or digest):
+     There are two levels of summary data:
+
+     - **Tag/digest-level** - for every namespace, repository, and reference (tag or digest):
+       - Unique IP address count
+       - Pulls by tag count
+       - Pulls by digest count
+       - Version check count
+     - **Repository-level** - for every namespace and repository:
```

The data points for repo-level and tag-level are both the same, right? So I don't think we need to describe them twice. I'm aligning the descriptions here with what I put in the other doc.

```suggestion
     There are two levels of summary data:

    - Repository-level, a summary of every namespace and repository
    - Tag- or digest-level, a summary of every namespace, repository, and reference
      (tag or digest)
```